src/components/DlToastMessage/helpers/render.ts

```diff
@@ -0,0 +1,54 @@
+import * as VueDemi from 'vue-demi'
```

Comment:
Please try to not use default imports in files like these which only require 2 of the full exported modules
import isVue3 or isVue2 straight from vue-demi

Reply:
I need a 'render' from vue 3, but it is not available in vue 2

Reply:
if i do "import {render} from 'vue-demi'" then i will have errors in vue 2

![image](https://user-images.githubusercontent.com/45905083/226927824-fa3387ba-26a5-42c4-ba35-3a5572b90228.png)